commerce-cloud-code/app_yourshophere/cartridge/api/Form.js

```diff
@@ -90,4 +91,19 @@ Form.prototype.rowValues = function (businessObject, parameterMap) {
     return rows;
 };
 
+Form.prototype.temp = function (parameterMap) {
+    const CacheMgr = require('dw/system/CacheMgr');
+    let tempObject = { custom: {} };
+    tempObject = this.persist(tempObject, parameterMap);
+    const tempCache = CacheMgr.getCache('Form');
```

Comment:
I'd prefer
`CacheMgr.getCache('Form').get(this.name + session.sessionID, (map) => this.persist(tempObject, parameterMap))`